snmpd/run.sh

```diff
@@ -39,24 +41,40 @@ extend hass_supervisor_version '/usr/bin/bashio /bashio_info.sh supervisor'
 extend hass_state '/usr/bin/bashio /bashio_info.sh state'
 extend hass_supported '/usr/bin/bashio /bashio_info.sh supported'
 
-#libreNMS distro detection
+# lldpd
+master agentx
```

Comment:
This line duplicates the same setting above at line 22/24

Reply:
you're right, nice catch :+1: